src/Card/CardTitle.js

```diff
@@ -49,36 +49,22 @@ export const CardTitle = ({
   return (
     <div {...getProps(onClick)}>
       {
-        (typeof title === 'string') &&
-          <h2>
-            {icon}
-            {title}
-          </h2>
+        (typeof title === 'string') && <h2> {icon} {title} </h2>
```

Comment:
Maybe you can add `(` and break the line? I don't know it looks difficult to read in a single line
```suggestion
(typeof title === 'string') && (
  <h2>{icon}{title}</h2>
)
```